opentelemetry-sdk/src/logs/log_emitter.rs

```diff
@@ -83,19 +93,33 @@ impl opentelemetry::logs::LoggerProvider for LoggerProvider {
 impl LoggerProvider {
     /// Create a new `LoggerProvider` builder.
     pub fn builder() -> Builder {
+        otel_info!(
+            name: "logger_provider_builder"
+        );
         Builder::default()
     }
 
     pub(crate) fn log_processors(&self) -> &[Box<dyn LogProcessor>] {
+        otel_debug!(
```

Comment:
It should be good enough to get the processor count information just once during LoggerProvider build. Is there any value in repeatedly logging it?

Reply:
to unblock and merge this PR quickly, I think its best to remove all internal logging that are not error/warning, and we can selective bring them one by one in future PRs.